logstash-core/lib/logstash/logging/slow_logger/freq_items.rb

```diff
@@ -0,0 +1,77 @@
+# encoding: utf-8
+require "logstash/namespace"
+
+module LogStash; module Logging; module Util
+
+    class FreqItems
+
+      class Statistics
+        attr_reader :mean, :min, :max
+        def initialize
+          @n = 0
+          @mean = 0.0
+          @m2 = 0.0
+          @min = -1
+          @max = 0
+        end
+
+        def update(x)
+          @n += 1
```

Comment:
Can you add comments here. Looking at the code, I don't know what you are updating and what the calculations are?